.github/workflows/code-vulnerability-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: "Code Vulnerability Analysis"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '0 7 * * *'
+    
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go' ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v2
+      with:
+        languages: go
+        
+    - name: Build code
+      run:
+       touch bootstrap
+       make release
```

Comment:
What is the `touch bootstrap` for?

There is no rule called `release` in the [Makefile](https://github.com/onflow/cadence-tools/blob/master/Makefile). Maybe use `test` instead.

```suggestion
       touch bootstrap
       make test
```

Reply:
should we have a top level Makefile that builds all or should we create per tool actions? I fear the current action won't build the tools.

Reply:
@turbolent Thanks for having a look and changing from release to test. Touch bootstrap was just a place holder because when I tried doing a custom build it complained about not having a valid "Bootstrap" file.

@sideninja If we had a common makefile which "autobuild" could grab that would be fine. I tested this in a dummy repo and it works which is why I asked @turbolent to check it as well.